### PR TITLE
Sync Maven and Ant plugin configuration

### LINKED ISSUE #1165: Documentation around ant based sql code generation needs some love
I'm using gradle so I'm using the ant codegen stuff. Three things in particular tripped me up.

1. The `querydsl-sql-codegen` module should be mentioned since that's where the `AntMetaDataExporter` resides.
2. The parameters for `com.mysema.query.sql.ant.AntMetaDataExporter` are not the same as the maven parameters. `dbDriverClass` instead of `jdbcDriver`, `dbUrl` instead of `jdbcUrl`, etc.
3. The parameter `targetSourceFolder` is required. A `NullPointerException` is thrown without it.

I've opened a few bugs now but I haven't said thanks to you guys for this project. It's going to be super helpful in my projects and it works really well once I've figured some things out.

### PR DESCRIPTION
**Pull request by timowest**
Fixes #1165

### CODE REVIEW

**Comment by Shredder121** [comment]
[Tutorial: Writing Tasks](http://ant.apache.org/manual/tutorial-writing-tasks.html) doesn't mention `@parameter` doclet tags, so I think these do nothing.

File: querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ant/AntMetaDataExporter.java [hunk 1]
```diff
@@ -37,190 +38,250 @@
...
+
 
     @Override
     @SuppressWarnings({ "unchecked", "rawtypes" })
     public void execute() {
         Connection dbConn = null;
-        File targetPackagePath = new File(targetSourceFolder);
+        File targetPackagePath = new File(targetFolder);
+
+        if (targetFolder == null) {
```
Comment (Shredder121) [inline querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ant/AntMetaDataExporter.java]: You should move this null check to before you use the variable.

**Comment by timowest** [comment]
Made the required changes.

**Comment by Shredder121** [comment]
there are a few occurrences of suffix changed to prefix.

**Comment by timowest** [comment]
Fixed those as well.

**Comment by timowest** [comment]
@Shredder121 Is this now good to merge?

**Comment by Shredder121** [comment]
I believe so yes.

**Pull request merged**